app/controllers/chat_controller.rb

```diff
@@ -340,6 +340,17 @@ def dismiss_retention_reminder
     render json: success_json
   end
 
+  def quote_messages
+    params.require(:message_ids)
+
+    @chat_channel = ChatChannel.find(params[:chat_channel_id])
```

Comment:
This will raise a `ActiveRecord::RecordNotFound` error if the param is invalid. We usually do a `find_by` and return the right response if we don't get anything back.

Reply:
ah yeah you are right, my bad